internal/run/consumer.go

```diff
@@ -146,14 +146,16 @@ func (c *Consumer) Consume(log *zap.Logger, repository *model.Repository) (err e
 		annotations["effx.io/file-path"] = effxYAMLFile
 
 		if lang != "" {
-			tags["lang"] = lang
+			annotations["effx.io/vcs-connect-inferred-tags"] = "language"
```

Comment:
im curious if we need this annotation to be the same for effx-cli and vcs connect? otherwise frontend will need to parse 2 different ones. not a huge deal though.

Reply:
i'm using `effx.io/inferred-tags` that cool?

Reply:
sure